docker/run/Dockerfile

```diff
@@ -93,6 +96,12 @@ RUN \
     flutter config --no-analytics && \
     flutter doctor && \
     pip install conan==$CONAN_VERSION && \
+    # Install golong in order to have `go mod` as package manager.
+    curl -s https://dl.google.com/go/go$GO_VERSION.linux-amd64.tar.gz -o go.tgz && \
```

Comment:
What's the reason to not use an `apt` package here?

Reply:
I wanted to use this specific and fixed version, but maybe that was not the right reason as one could specify that also to apt. Want me to change it?

Reply:
Well, *if* the version you want is available via `apt` then yes, I'd prefer to have that changed. But if the version is not available via `apt` the code should stay as-is, but I'd appreciate a comment in the commit message that this particular version is not available via `apt`, and thus we have to use this less preferred (as more complicated and not package-managed) way of installing the software.

Reply:
Ok, thanks! The apt sources only have:
```
golang/eoan 2:1.12~1ubuntu1 amd64
golang/eoan 2:1.12~1ubuntu1 i386
```

Reply:
Actually, the Docker image uses Debian Sid, [which has](https://packages.debian.org/sid/golang) `golang-1.13` if that makes a difference.

Reply:
Thanks! Yeah my mistake, I hadn't notice that my `docker run -it  ... /bin/bash` failed so I ran `apt list` on my host machine without noticing.